src/TypeScriptGenerator.ts

```diff
@@ -593,8 +587,12 @@ function getRefTypeName(name: string): string {
   return `${name}$ref`;
 }
 
+// Should match FLOW_TRANSFORMS array
+// https://github.com/facebook/relay/blob/master/packages/relay-compiler/language/javascript/RelayFlowGenerator.js#L660
```

Comment:
Best to not link to a branch, as that can go out of date:

```suggestion
// https://github.com/facebook/relay/blob/v4.0.0/packages/relay-compiler/language/javascript/RelayFlowGenerator.js#L621-L627
```